webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+// The path to the ceisum source code
+const cesiumSource = 'node_modules/cesium/Source';
+const cesiumWorkers = '../Build/Cesium/Workers';
+
+module.exports = [{
+	context: __dirname,
+	entry: {
+		app: './src/index.js'
+	},
+	output: {
+		filename: '[name].js',
+		path: path.resolve(__dirname, 'dist'),
+		sourcePrefix: '' // Needed by Cesium for multiline strings
+	},
+	resolve: {
+		alias: {
+			// Cesium module name
+			cesium: path.resolve(__dirname, cesiumSource)
+		}
+	},
+	amd: {
+		// Enable webpack-friendly use of require in cesium
+		toUrlUndefined: true
+	},
+	module: {
+		rules: [{
+			test: /\.css$/,
+			use: [ 'style-loader', 'css-loader' ]
+		}, {
+			test: /\.(png|gif|jpg|jpeg|svg|xml|json)$/,
```

Comment:
We could use [url-loader](https://github.com/webpack-contrib/url-loader) for images

Reply:
True. It looks like the file-loader re-emits as well as returns the filename, which we don't need to do since we're copying over the assets.

Reply:
Maybe leave file loader for svg, xml and json?

Reply:
I was able to load in json and svg files just fine with the url-loader. If it's a concern, we can make the test only include files in the `Widgets` or `Assets` directories, as that's where the one's we're serving are coming from.

Reply:
Is it possible that json, xml and svg files are large enough so that they can benefit from the DataURL conversion of url-loader? If not, then I would not bother test them with file-loader.